{"uid":"4b4757e66a1912dae1a509f688f20b0f","children":[{"name":"Product defects","children":[{"name":"Timed out retrying after 4000ms: Expected to find content: '필수 정보입니다.' but never did.","children":[{"name":"0. 사이트 개설","uid":"7d7f6607efd7119b","parentUid":"a511ba7a8b408ecf840e47900f727299","status":"failed","time":{"start":1666158434924,"stop":1666158464845,"duration":29921},"flaky":false,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]}],"uid":"a511ba7a8b408ecf840e47900f727299"}],"uid":"8fb3a91ba5aaf9de24cc8a92edc82b5d"},{"name":"Test defects","children":[{"name":"The following error originated from your application code, not from Cypress.\n\n  > $ is not defined\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","children":[{"name":"5-1. class insert(강의 라이브 생성))","uid":"22f52875ddb4d3d0","parentUid":"6e767925521d7b938b306c13593e5caf","status":"broken","time":{"start":1666158748552,"stop":1666158759861,"duration":11309},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":true,"retriesCount":0,"retriesStatusChange":false,"parameters":[]}],"uid":"6e767925521d7b938b306c13593e5caf"}],"uid":"bdbf199525818fae7a8651db9eafe741"}],"name":"categories"}